virtdata-userlibs/src/main/resources/docs-for-virtdata/bindings/funcref_conversion.md

```diff
@@ -228,6 +228,7 @@ Convert the input value to a long.
 
 
 ## ToMD5ByteBuffer
+**[ DEPRECATED ]** no longer supported use DigestToByteBuffer with stronger digest algorithm.
```

Comment:
We should make the docs exporter do this for us in a separate issue.

Reply:
I created a new docs issue to track that effort.